examples/AsyncDisplayKitOverview/Sample/OverviewComponentsViewController.m

```diff
@@ -109,18 +134,17 @@ - (void)viewDidLoad
     self.title = @"AsyncDisplayKit";
     
     [self setupData];
-    [self setupTableNode];
 }
 
 - (void)viewWillAppear:(BOOL)animated
 {
     [super viewWillAppear:animated];
     
-    ASTableView *tableView = self.tableNode.view;
-    [tableView deselectRowAtIndexPath:tableView.indexPathForSelectedRow animated:YES];
+    [_tableNode.view deselectRowAtIndexPath:_tableNode.view.indexPathForSelectedRow animated:YES];
```

Comment:
Can we check if there is a tableNode version of this method?  Can you call deselect directly on the tableNode?